_posts/2025-08-26-strimzi-metrics-reporter.md

```diff
@@ -0,0 +1,97 @@
+
+
+The `Strimzi Metrics Reporter` exposes metrics in the Prometheus format directly over HTTP, significantly simplifying the setup and reducing resource consumption.
+It is also a standalone MetricsReporter plugin that can be used with any Kafka Client deployment. 
+
+
+Here, we will discuss Strimzi Metrics Reporter installation and configuration, and guide you on how to use it effectively to monitor your Kafka clusters.
+
+
+## Key Features
+* Direct Prometheus Integration: Exposes Kafka metrics directly to Prometheus through an HTTP endpoint.
+* Configurable Metrics Collection: Allows users to specify which metrics should be collected using a flexible allowlist.
+
+### Deploying Metrics Reporter
+The first step is to install the metrics reporter in your `Kafka Custom Resource`. 
+To do this, add the following:
+
+```yaml
+apiVersion: kafka.strimzi.io/v1beta1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+  # ...
+    metricsConfig:
+      type: strimziMetricsReporter
+  # ...  
+```
+
+By adding `type: strimziMetricsReporter` to the `metricsConfig` section of your Kafka CR, `Strimzi` will export a sensible set of default metrics.
+However, you can add you own custom values (providing the metrics are available) by filtering the metrics by name. 
+This is achieved by adding another field called `values` and within that field, adding `allowList`.
+Then you can specify which metrics you want to collect as a regex list, where each entry is used to filter allowed metrics. 
+For example:
+
+```yaml
+apiVersion: kafka.strimzi.io/v1beta1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+  # ...
+    metricsConfig:
+      type: strimziMetricsReporter
+      values:
+        allowList:
+          - "kafka_log.*"
+          - "kafka_network.*"
+  # ...  
+```
+
+If you are adding Metrics Reporter to an existing cluster (instead of starting a fresh cluster) then you will see all your Kafka broker and Controller pods roll so that the metrics reporter config can be added to your Kafka CR.
+
+If you would like to deploy your Kafka cluster with the Metrics Reporter enabled from the start, you can use the examples in our [Strimzi Metrics Reporter Examples folder](https://github.com/strimzi/strimzi-kafka-operator/tree/main/packaging/examples/metrics/strimzi-metrics-reporter) by running the following command from directory:
+
+```bash
+$ kubectl apply -f kafka-metrics.yaml -n myproject
+```
+
+We have also provided some [Grafana Dashboards](https://github.com/strimzi/strimzi-kafka-operator/tree/main/packaging/examples/metrics/strimzi-metrics-reporter/grafana-dashboards) in the same directory to help you get started with visualizing your Kafka metrics in the Prometheus format.
+
+## Conclusion
+We have shown you how to deploy the `Strimzi Metrics Reporter` in your Kafka cluster and given some compelling reasons why you might want to use it in preference to the previously recommended `JMX Exporter`.
```

Comment:
Here but also in other places - project names are not code. If you want to make them look differently, italics would probably look muhc better than fixed size font.